app-crypt/swtpm/swtpm-0.2.0.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit autotools python-any-r1
+
+DESCRIPTION="Libtpms-based TPM emulator"
+HOMEPAGE="https://github.com/stefanberger/swtpm"
+SRC_URI="https://github.com/stefanberger/swtpm/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="fuse gnutls test seccomp"
+
+COMMON_DEPEND="dev-libs/libtpms
+	dev-libs/openssl:0
+	fuse? (
+		dev-libs/glib:2
+		sys-fs/fuse:0
+	      )
+	gnutls? (
+		   dev-libs/libtasn1:=
+		   >=net-libs/gnutls-3.1.0[tools]
+		)
+	seccomp? ( sys-libs/libseccomp )
+"
+
+DEPEND="${COMMON_DEPEND}
+	test? ( net-misc/socat:=
```

Comment:
Nope, socat doesn't have a subslot. He probably meant that you should split this into multiple lines, like you do with gnutls. Ie, 
```
gnutls? (	
	dev-libs/libtasn1:=
  	>=net-libs/gnutls-3.1.0[tools]
)
```

Reply:
Ok whitespace added